app/src/routes/v1/docs.js

```diff
@@ -16,6 +17,17 @@ function getSpec() {
   return spec;
 }
 
+router.use(
+  helmet({
+    contentSecurityPolicy: {
+      directives: {
+        'img-src': ['data:', 'https://cdn.redoc.ly'],
+        'script-src': ['blob:', 'https://cdn.redoc.ly']
```

Comment:
I think you need to add back the defaults when you do an override (eg: `self`) to both of these.
might fix those CORS errors when viewing the api spec